README.md

```diff
@@ -1,27 +1,50 @@
-Ubuntu documentation service
-===
+# Ubuntu documentation service
 
 [![Build Status](https://travis-ci.org/ubuntudesign/docs.ubuntu.com.svg?branch=master)](https://travis-ci.org/ubuntudesign/docs.ubuntu.com)
 
 A central platform for hosting Ubuntu's various documentation portals.
 
-Basic usage
----
+## Local development
+
+### Dependencies
+
+1. [Install Docker](https://www.docker.com/community-edition#/download)
+2. Get [the `repo` command](https://github.com/esrlabs/git-repo), e.g.:
```

Comment:
It works mostly fine... Just trouble configuring DNS at work and existing solutions don't seem to work.
Works with no problems at home.